Tests/test_PDBList.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2016 by Jacek Smietanski.  All rights reserved.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+"""Testing access to the PDB over the internet."""
+
+import contextlib
+import os
+import shutil
+import tempfile
+import unittest
+
+import requires_internet
+requires_internet.check()
+
+# We want to test this module:
+from Bio.PDB.PDBList import PDBList
+
+
+class TestPBDListGetList(unittest.TestCase):
+    """
+    Test methods responsible for getting lists of entries.
+    """
```

Comment:
Can you write this as one line please, as per PEP257 https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

i.e.

```
 """Test methods responsible for getting lists of entries."""
```

(As the same for the rest of the file)